autoPlay.js

```diff
@@ -50,7 +55,7 @@ if (thingTimer){
 function firstRun() {
 	// disable particle effects - this drastically reduces the game's memory leak
 	if (g_Minigame !== undefined) {
-		g_Minigame.CurrentScene().DoClickEffect = function() {};
+	//	g_Minigame.CurrentScene().DoClickEffect = function() {}; //this is irritating
```

Comment:
This is in place to minimize the game's horrible memory leak. By all means disable it in your version, but the main branch is going to leave this in.